default-plugins/status-bar/src/main.rs

```diff
@@ -136,7 +136,9 @@ fn color_elements(palette: Palette) -> ColoredElements {
 }
 
 impl ZellijPlugin for State {
-    fn load(&mut self) {
+    type Options = ();
+
+    fn load(&mut self, _: ()) {
```

Comment:
Stylistically, it might be preferable to use `Self::Options` instead of `()` here? Then you'd only need to change the configuration type in one place (line 139) instead of there and this line!